fastai/basic_data.py

```diff
@@ -272,6 +272,19 @@ def sanity_check(self):
             warn(message)
             print(final_message)
 
+    def slice(self, slice_train:slice, slice_valid:slice=slice(None), slice_test:slice=slice(None)):
+        "Return sliced version of `self`, for quick experimenting with massive datasets"
+        dls = []
+        for ds, slc in zip((self.train_ds, self.valid_ds, self.test_ds), (slice_train, slice_valid, slice_test)):
+            if ds is None: dls.append(None)
+            else: dls.append(DataLoader(ds[slc], batch_size=self.batch_size))
```

Comment:
There are plenty of other arguments to pass to `DataLoader`, and it could also be a subclass of a `DataLoader` here.

Reply:
Thank you for the reply, the `use_partial_data` is what I wanted.

I've watch the full courses at https://course.fast.ai/ but found no relating contents.
please publish a reading list, so that new users can get familiar enough to known such features.

Reply:
This function is an external contribution, which was added after the course material was prepared, that's why it's not mentioned there. The [docs](https://docs.fast.ai/data_block.html#ItemList.use_partial_data) are the best place to get information though it's true this feature could be better advertised.

Closing the PR if you're satisfied with `use_partial_data` for now. If you want to update it with the changes I mentioned, I'd be happy to merge it.